pydicom/pixel_data_handlers/numpy_handler.py

```diff
@@ -104,19 +104,7 @@ def get_pixeldata(dicom_dataset):
 
     pixel_bytearray = dicom_dataset.PixelData
 
-    if dicom_dataset.BitsAllocated == 1:
-        # if single bits are used for binary representation, a uint8 array
-        # has to be converted to a binary-valued array (that is 8 times bigger)
-        try:
-            pixel_array = numpy.unpackbits(
-                numpy.frombuffer(pixel_bytearray, dtype='uint8'))
-        except NotImplementedError:
-            # PyPy2 does not implement numpy.unpackbits
-            raise NotImplementedError(
```

Comment:
Could you fix up the failing unit test, too?

Reply:
It can just be removed, likewise the skip condition on `OneBitAllocatedTests`.

Reply:
Done (if I understood correctly how things were expected to work ...)

Reply:
Hmm, interesting - don't understand why CI of [the commit](https://github.com/pydicom/pydicom/pull/627/commits/91b8f88d471d2d1b553a7181ecace28fc7e819fc) prior to removing those tests succeeded.